sympy/functions/elementary/trigonometric.py

```diff
@@ -2950,9 +2948,9 @@ def _eval_aseries(self, n, args0, x, logx):
         if args0[0] is S.Infinity:
             return (pi/2 - acot(1/self.args[0]))._eval_nseries(x, n, logx)
         elif args0[0] is S.NegativeInfinity:
-            return (pi*Rational(3, 2) - acot(1/self.args[0]))._eval_nseries(x, n, logx)
```

Comment:
Sorry I took some time to get back. I like the changes introduced for `atan` and that looks good to me but if you think about it all changes for `acot` can be supported through `atan` (as they have rewrites for each other). Hence I would something minimalistic like this would do 
```
    def _eval_aseries(self, n, args0, x, logx):
        if args0[0] in [S.Infinity, S.NegativeInfinity]:
            return atan(1/self.args[0])._eval_nseries(x, n, logx)
        else:
            return super()._eval_aseries(n, args0, x, logx)
```
I did a basic check and I think everything works. Just check again and get back to me !